packages/components/src/components/breadcrumb/breadcrumb-item.tsx

```diff
@@ -0,0 +1,79 @@
+import { Component, h, Element, State, Listen } from '@stencil/core';
+
+@Component({
+  tag: 'ifx-breadcrumb-item',
+  styleUrl: 'breadcrumb-item.scss',
+  shadow: true,
+})
+export class BreadcrumbItem {
+
+  @State() isLastItem: boolean = false;
+  @Element() el;
+
+  @Listen('mousedown', { target: 'document' })
+  handleOutsideClick(event: MouseEvent) {
+    const path = event.composedPath();
+    if (!path.includes(this.el)) {
+      this.closeDropdownMenu();
+    }
+  }
+
+  getDropdownMenu() {
+    const dropdownMenu = this.el.shadowRoot.querySelector('.dropdown-menu');
+    return dropdownMenu
+  }
+
+  getMenuIconWrapper() { 
+    const menuIconWrapper = this.el.querySelector('ifx-breadcrumb-item-label')
+    const container = menuIconWrapper.shadowRoot.querySelector('.breadcrumb-item-label-container')
+    const menuWrapper = container.querySelector('.menu-icon-wrapper')
+    return menuWrapper
+  }
+
+  handleClassList(el, type, className) {
+    el.classList[type](className)
+  }
+
+  closeDropdownMenu() {
+    const dropdownMenu = this.getDropdownMenu()
+    const menuWrapper = this.getMenuIconWrapper()
+    this.handleClassList(dropdownMenu, 'remove', 'open')
+    this.handleClassList(menuWrapper, 'remove', 'show')
+  }
+
+  toggleDropdownMenu() { 
+    const dropdownMenu = this.getDropdownMenu()
+    const menuWrapper = this.getMenuIconWrapper()
+    this.handleClassList(dropdownMenu, 'toggle', 'open')
+    this.handleClassList(menuWrapper, 'toggle', 'show')
+  }
+
+  componentWillLoad() { 
+    const breadcrumbItems = this.el.closest('ifx-breadcrumb').querySelectorAll('ifx-breadcrumb-item')
```

Comment:
Handling last breadcrumb item: You're determining if the current item is the last breadcrumb item in the componentWillLoad lifecycle hook. However, if the list of breadcrumbs changes after the component is loaded, this will not update as expected. This could lead to incorrect behavior.

Reply:
What is a user case in which the number of breadcrumb items changes without page refresh?

Reply:
For example in an SPA we just add a new item

so we iterate over all items in a list and add a breadcrumb for each of them. Then we add an item to the items list. Or remove one.